generator/vulkan/build_integration.zig

```diff
@@ -73,7 +67,29 @@ pub const GenerateStep = struct {
         const self = @fieldParentPtr(GenerateStep, "step", step);
         const cwd = std.fs.cwd();
 
+        var man = self.builder.cache.obtain();
+        defer man.deinit();
+
         const spec = try cwd.readFileAlloc(self.builder.allocator, self.spec_path, std.math.maxInt(usize));
+        // TODO: Look into whether this is the right way to be doing
+        // this - maybe the file-level caching API has some benefits I
+        // don't understand.
+        man.hash.addBytes(self.spec_path);
+        man.hash.addBytes(spec);
+        man.hash.addBytes(self.output_name);
```

Comment:
Perhaps we can remove this one also. I don't think that its necessary to let the user pass in the cache-relative directory for vk.zig. Thats something for another time, though